autogluon/training/docker/1.1/py3/cu118/Dockerfile.gpu

```diff
@@ -0,0 +1,66 @@
+ARG CUDA_VER=11.8
+ARG PYTHON_VERSION=3.10.8
+
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:2.0.1-gpu-py310-cu118-ubuntu20.04-sagemaker
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+RUN mamba install pytorch=2.1.0 --strict-channel-priority --override-channels -c https://aws-ml-conda.s3.us-west-2.amazonaws.com -c nvidia/label/cuda-11.8.0 -c conda-forge -y --debug
```

Comment:
do we still need `--debug` here? Also, shall we remove the old version first to avoid image size incresae?
```
# Uninstall the existing version
RUN mamba remove pytorch --force -y
```

Reply:
we don't need debug, had added it to see the logs while building the image. made the necessary changes, now waiting for a hotfix from the DLC team to proceed running sanity checks